proto/viam/app/mlinference/v1/ml_inference.proto

```diff
@@ -20,4 +22,9 @@ message GetInferenceRequest {
   string organization_id = 4;
 }
 
-message GetInferenceResponse {}
+message GetInferenceResponse {
+  viam.service.mlmodel.v1.FlatTensors output_tensors = 1;
+  viam.app.data.v1.Annotations annotations = 2;
+  // anything else you want to say
+  google.protobuf.Struct extra = 99;
```

Comment:
I think there's pretty low risk of needing to deprecate this field, but I would hate to introduce a backwards-incompatible breaking change over something as minor as an extra field. Is this intended to be used for something immediately? If not, I would err on just adding it as needed. WDYT?

Reply:
OK! That makes sense to me. Just thought it might be helpful because I find myself wanting to pass more info around in the MLModel in delphi. But we can add it if we need.

Reply:
If we wanted to pass more info around in the MLModel in Delphi, wouldn't we then be working with the ML model proto? https://github.com/viamrobotics/api/blob/main/proto/viam/service/mlmodel/v1/mlmodel.proto

Reply:
Yeah, fixed it up! Thanks for this!